semantic-conventions/src/opentelemetry/semconv/templating/markdown.py

```diff
@@ -399,6 +399,12 @@ def _render_table(self, semconv, parameters, output):
                 continue
             if self.render_ctx.is_full or attr.is_local:
                 attr_to_print.append(attr)
+        if self.render_ctx.group_key is not None and not attr_to_print:
```

Comment:
Why don't we just:

```suggestion
        if not attr_to_print:
```

Reply:
because there could be the case where a semantic convention extends an existing one without introducing any new attribute but only constraints. An example of this is `faas.http`.

Reply:
Hmm, but how is that related to the group_key? Isn't that the filter key like `tech-specific` or `connection-level`?

Reply:
yes! If you filter over a key and you don't get any result, you will receive a warning.

Reply:
OK, I think I understand now. The problem is that `attr_to_print` being empty is fine if you also have (any-of) constraints, unless you added a filter, because then you probably meant to filter out only some but not all attributes.